proof-producer/bin/proof-producer/include/nil/proof-generator/prover.hpp

```diff
@@ -541,6 +545,25 @@ namespace nil {
                 return true;
             }
 
+            bool save_binary_assignment_table_to_file(const boost::filesystem::path& output_filename) {
+                BOOST_LOG_TRIVIAL(info) << "writing binary assignment table to " << output_filename;
+                
+                if (!assignment_table_.has_value() || !table_description_.has_value()) {
+                    BOOST_LOG_TRIVIAL(error) << "no assignment table is currently loaded into the Prover";
```

Comment:
`No assignment table is currently loaded`

Reply:
fixed